docs/DeveloperGuide.md

```diff
@@ -903,6 +929,49 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
+### Adding an applicant
+
+1. Adding an applicant while all applicants are being shown in the *GUI*.
+
+    1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list. No other applicant named Alice in the list.
+       Bob is an applicant in the list.
+
+    2. Test case: 
+       ```
+       add 
+       n/Alice 
+       p/99990000 
+       e/alice@mail.com 
+       j/Software Engineer
+       r/Technical Interview
+       s/Java
+       ```
+       Note: The above code block is meant to be input into one line.<br>
```

Comment:
I think no need to repeat, just say once that it applies for the rest in this section.

Use the HTML Note block.

Also, should it be `meant to be inputted`?